data_processing.py

```diff
@@ -0,0 +1,76 @@
+import os
```

Comment:
Maybe add a short comment describing what this script does? `data_processing` is not quite as descriptive as `check_increasing_timestamp`.